backend/controllers/Auth/RegisterController.js

```diff
@@ -9,7 +9,7 @@ class RegisterController {
 
         const user = await this.userService.register(email, password);
 
-        const registeredUser = await this.userService.findById(user.id);
+        const registeredUser = await this.userService.getUserByIdWithExcludedPassword(user.id);
```

Comment:
Default should be excluded, if you need include it you should put argument. 

So please don't create such crazy names.

this.userRepository.getById(id);

Also don't write "getUserById" if you are using userService or userRepository. 
By default it means you are getting user from userRepository, contract from contractRepository.

Reply:
userService.getById(id, {include: ['password']}); 

something like this

Reply:
done